lib/packages-dependencies.php

```diff
@@ -1,265 +1,5 @@
 <?php
 /**
- * Returns array with the list of WordPress packages and their dependencies.
- *
+ * @deprecated since 5.6. Script dependencies should be read from generated `build/[package]/index.deps.json`.
```

Comment:
I've had experience with plugins that avoid removing PHP files because it's caused problems with some hosts with unusual setups. Is that a concern here? Should this file be removed completely?

Reply:
We never did it before. I think it's fine to remove.

Reply:
Removed, that fixes one of the lint errors.